framework/src/Volo.Abp.BlobStoring.Minio/Volo/Abp/BlobStoring/Minio/MinioBlobProvider.cs

```diff
@@ -0,0 +1,159 @@
+                return true;
+            }
+
+            return false;
+        }
+
+        public override async Task<bool> ExistsAsync(BlobProviderExistsArgs args)
+        {
+            var blobName = MinioBlobNameCalculator.Calculate(args);
+
+            return await BlobExistsAsync(args, blobName);
+        }
+
+        public override async Task<Stream> GetOrNullAsync(BlobProviderGetArgs args)
+        {
+            var blobName = MinioBlobNameCalculator.Calculate(args);
+
+            if (!await BlobExistsAsync(args, blobName))
+            {
+                return null;
+            }
+            try
+            {
+                var client = GetMinioClient(args);
+             
+               var stat = await client.StatObjectAsync(GetContainerName(args), blobName);
+
+                MemoryStream returnStream = new MemoryStream();
+
+                await client.GetObjectAsync(GetContainerName(args), blobName,
+                                                 (stream) =>
+                                                 {
+                                                     if (stream != null)
+                                                     {
+                                                         stream.CopyTo(returnStream);
+
+                                                            // returnStream = new MemoryStream(stream.GetAllBytes());
+
+                                                     }
+                                                 });
+
+                return returnStream;
+            }
+            catch (MinioException ex)
+            {
+
+            }
+            return null;
+
+        }
+
+
+        private MinioClient GetMinioClient(BlobProviderArgs args)
+        {
+            var configuration = args.Configuration.GetMinioConfiguration();
+            var client = new MinioClient(configuration.EndPoint, configuration.AccessKey, configuration.SecretKey);
+            if (configuration.WithSSL)
+            {
+                client.WithSSL();
+            }
+
+            return client;
+
+        }
+
+
+
+        protected virtual async Task CreateBucketIfNotExists(BlobProviderArgs args)
+        {
+            var client = GetMinioClient(args);
+            var containerName = GetContainerName(args);
+            if (!await client.BucketExistsAsync(containerName))
+            {
+                await client.MakeBucketAsync(containerName);
+            }
+
+        }
+
+        private async Task<bool> BlobExistsAsync(BlobProviderArgs args, string blobName)
+        {
+            // Make sure Blob Container exists.
+            if (await ContainerExistsAsync(args))
+            {
+                try
+                {
+                    await GetMinioClient(args).StatObjectAsync(GetContainerName(args), blobName);
+                    return true;
+                }
+                catch (MinioException ex)
```

Comment:
Why hide exception?
Developers never get exceptions,I think this is not a good practice. 
Just throw, that's fine.